modules/client-configuration/pages/salt-client-proxy.adoc

```diff
@@ -19,7 +19,12 @@ For information about creating a bootstrap repository, see xref:client-configura
 By default, the SSH  port is [systemitem]``22``.
 . In the [guimenu]``User`` field, type the username to log in to the client.
 By default, the username is [systemitem]``root``.
-. In the [guimenu]``Password`` field, type password to log in to the client.
+. In the [guimenu]``Authentication Method`` field, select the authentication method to use for bootstrapping the client.
++
+* For password authentication, in the [guimenu]``Password`` field, type password to log in to the client.
+* For SSH Private key authentication, enter the private key and the associated passphrase.
+The key is only stored for as long as the bootstrapping process takes to complete.
+The key is deleted after bootstrapping is successful.
```

Comment:
Nitpick: the key is also deleted, when the bootstrapping is unsuccessful :-), so either I'd suggest using something more general ("..the key is deleted after bootstrapping"), or I'd even leave this sentence out (the line 26 already says that).